apps/browser/src/components/Chat/Chat.tsx

```diff
@@ -207,7 +207,9 @@ const Chat: React.FC<ChatProps> = ({ evo, onMessage, messages, goalEnded }: Chat
       <div className="Messages">
       {messages.map((msg, index) => (
           <div key={index} className={`MessageContainer ${msg.user}`}>
-            <div className="SenderName">{msg.user.toUpperCase()}</div>
+            {index === 0 || messages[index - 1].user !== msg.user ? (
```

Comment:
Works as a temporary solution, but we're gonna need something more robust once we have non-cont mode